levels/utils.go

```diff
@@ -25,7 +25,7 @@ func print_line(text string, keypress chan []byte, echo_state bool) {
 	var counter = 0
 	var b []byte = make([]byte, 1)
 	for _, char := range text {
-		print(string(char))
+		fmt.Printf("%s", string(char))
```

Comment:
@Foxiy I assume this was necessary but seems like a win anyway, since we were already using `fmt` throughout the file.